### PR TITLE
Update from source: Drag drop HDR files; some CSS fixes

### PR DESCRIPTION
**Pull request by stevesan**

### CODE REVIEW

File: packages/space-opera/src/components/ibl_selector/ibl_selector.ts [hunk 1]
```diff
@@ -164,15 +111,10 @@ export class IblSelector extends ConnectedLitElement {
       return;
     }
 
-    const filename = (files[0] as File).name;
-
-    const arrayBuffer = await files[0].arrayBuffer();
-    const safeObjectUrl = createSafeObjectUrlFromArrayBuffer(arrayBuffer);
-    const unsafeUrl = filename.match(/\.(hdr)$/) ?
-        safeObjectUrl.unsafeUrl + '#.hdr' :
```
Comment (mrdoob) [inline packages/space-opera/src/components/ibl_selector/ibl_selector.ts:172]: My hack is gone! 😢😁

Comment (elalish) [reply packages/space-opera/src/components/ibl_selector/ibl_selector.ts:172]: What hack?

Comment (stevesan) [reply packages/space-opera/src/components/ibl_selector/ibl_selector.ts:172]: Oh it's still there - I factored it into createBlobUrlFromEnvironmentImage - that what you mean?

Comment (mrdoob) [reply packages/space-opera/src/components/ibl_selector/ibl_selector.ts:172]: https://github.com/google/model-viewer/blob/07bd0edceff1f61f22fa3b0c1db3aae809392fb8/packages/space-opera/src/redux/lighting_dispatchers.ts#L43

Yesss! Definitely my best contribution to the project 😁

Comment (elalish) [reply packages/space-opera/src/components/ibl_selector/ibl_selector.ts:172]: ah, now I see; nice...

**Pull request merged**